src/sage/graphs/graph_decompositions/modular_decomposition.pyx

```diff
@@ -273,82 +316,30 @@ def __eq__(self, other):
             False
         """
         return (self.node_type == other.node_type and
-                self.node_split == other.node_split and
-                self.index_in_root == other.index_in_root and
-                self.comp_num == other.comp_num and
-                self.is_separated == other.is_separated and
                 self.children == other.children)
 
+    @classmethod
+    def create_leaf(cls, v):
+        """
+        Return Node object that is a leaf corresponding to the vertex ``v``
```

Comment:
add missing `.` at the end of the sentence.

Reply:
done in [451b325](https://github.com/sagemath/sage/pull/39038/commits/451b3256ac2cbbb00d2ff24ecc80339fe73d5e40)